api/Silk.Api/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Globalization;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using Silk.Api.Domain.Feature.Users;
+using Silk.Api.Helpers;
+using Silk.Api.Models;
+using Silk.Api.Services;
+
+namespace Silk.Api.Controllers
+{
+	[ApiController]
+	[Route("api/v1/[controller]")]
+	public class AuthenticationController : ControllerBase
```

Comment:
Can decorate controller with `Authorize` to protect all routes by default and only allow actions with `AllowAnonymous` to override auth checks